projects/coordinator/src/main/java/org/batfish/coordinator/resources/AuthenticationFilter.java

```diff
@@ -0,0 +1,65 @@
+package org.batfish.coordinator.resources;
+
+import java.security.AccessControlException;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.Provider;
+import org.batfish.common.CoordConsts;
+import org.batfish.common.Version;
+import org.batfish.coordinator.Main;
+
+/**
+ * This filter verify the access permissions for a user based on apiKey and version provided in
+ * request header.
+ */
+@Provider public class AuthenticationFilter implements ContainerRequestFilter {
```

Comment:
please enable the java formatting using the `google-java-format` plugin.

They had a [bug in the 1.3.0 version](https://github.com/google/google-java-format/issues/167#issuecomment-319532138) where it could disable itself – are you using the latest version?